src/components/elements/lockup/lockup.tsx

```diff
@@ -43,7 +43,7 @@ export const Lockup = ({
 
   if (!suLockupEnabled) {
     return (
-      <div className="py-10">
+      <div>
```

Comment:
we probably don't need the `div` here. Less markup is often a good way to go. But if we do need it, totally fine. I've seen some videos that suggest to remove elements if there's no attributes, or use fragments: `<>`